viewer/src/main/webapp/viewer-html/components/Edit.js

```diff
@@ -1062,6 +1062,8 @@ Ext.define("viewer.components.Edit", {
             allowBlank: !disallowNull,
             disabled: !this.allowedEditable(attribute),
             editable: !(attribute.hasOwnProperty('allowValueListOnly') && attribute.allowValueListOnly),
+            // typeAhead can only be enabled when editable is (also) true
+            typeAhead: true,
```

Comment:
We could also use the same value as `editable` `!(attribute.hasOwnProperty('allowValueListOnly') && attribute.allowValueListOnly),`